multi/stm32l4-multi/uart.c

```diff
@@ -35,29 +37,7 @@
 
 #define UART_CNT (UART1 + UART2 + UART3 + UART4 + UART5 + LPUART1)
```

Comment:
If LPUART is removed from reading/writing here, shouldn't be removed from this macros?

Reply:
It should be, fixed